bin/install-password-argon2

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o xtrace
+set -o errexit
+
+# If PHP < 7.2, exit
+if [[ $VERSION =~ ^master$ ]]; then
+	echo 'unknown PHP version; skip password-argon2 steps'
+	exit 0
+elif [ "$(printf "7.2\n$VERSION" | sort -V | head -n1)" < "7.2" ]; then
```

Comment:
Does this `if` statement work on your machine?

```sh-session
bash-4.4$ VERSION=foo; if [ $(printf "7.2\n$VERSION" | sort -V | head -n1) < "7.2" ]; then echo skip; fi
bash: 7.2: No such file or directory
bash-4.4$ VERSION=7.1; if [ $(printf "7.2\n$VERSION" | sort -V | head -n1) < "7.2" ]; then echo skip; fi
bash: 7.2: No such file or directory
```

It seems to me that `<` is interpreted as redirection, and bash is looking for a file named `7.2` instead.

Perhaps you want `[[]]`:

```sh-session
bash-4.4$ VERSION=7.2; if [[ "$(printf "7.2\n$VERSION" | sort -V | head -n1)" < "7.2" ]]; then echo skip; else echo go; fi
go
bash-4.4$ VERSION=7.1; if [[ "$(printf "7.2\n$VERSION" | sort -V | head -n1)" < "7.2" ]]; then echo skip; else echo go; fi
skip
bash-4.4$ VERSION=10; if [[ "$(printf "7.2\n$VERSION" | sort -V | head -n1)" < "7.2" ]]; then echo skip; else echo go; fi
go
bash-4.4$ VERSION=foo; if [[ "$(printf "7.2\n$VERSION" | sort -V | head -n1)" < "7.2" ]]; then echo skip; else echo go; fi
go
```

Reply:
Nope, you're right! Fixing!